app/controllers/api/v1/orders_controller.rb

```diff
@@ -23,7 +23,7 @@ class OrdersController < Api::V1::ApiController
           param :stockit_id, String, desc: "stockit designation record id", allow_nil: true
           param :beneficiary_id, String, allow_nil: true
           param :address_id, String
-          param :booking_type_id, String, desc: 'Booking type.(Online order or appointment)'
+          param :booking_type_id, String, desc: 'Booking type.(Online order or appointment)', allow_nil: true
```

Comment:
Is there a reason we need `allow_nil`?

Reply:
I think this is because `booking_type_id` is not required for international orders

cc: @shreyas098

Reply:
Yes